src/style.css

```diff
@@ -1,3 +1,96 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --header-height: 60px;
+  --font-size: 12px;
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: auto;
```

Comment:
Avoid fixing the container size with `height: auto;`. Let the content size dictate it. To avoid overflow or accidental scroll bar, you don't need to set the height property here unless specifically required.